ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -962,6 +976,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private
+- as: aws-usgov-ipi-private-sts-p3-f28-long-duration-cloud
```

Comment:
Job name should be consistent with workflow and end with`p3-f28-long-duration-cloud`, this one should be aws-ipi-disconnected-private-p3-f28-long-duration-cloud

Reply:
done